### ISSUE TITLE
Mauvais emplacement ?

### ISSUE THREAD

**Issue by EmmanuelBeziat** [idem-web-2016/site-manuel#1]
Salut,

Tu devrais plutôt garder ton site dans ton propre profil, plutôt que dans celui du groupe.

Tu peux sois cliquer sur "fork" pour copier directement celui-ci vers ton compte, sois recréer un repo directement.

Pense à supprimer celui-ci ensuite.

**Comment by aironpower** [comment]
Oui, j'ai pensé ça aussi ce matin quand j'ai vu tous les nouveaux membres.
Je vais le changer dès que j'arrive chez moi
